two_factor/forms.py

```diff
@@ -141,6 +142,9 @@ class DisableForm(forms.Form):
 class AuthenticationTokenForm(OTPAuthenticationFormMixin, Form):
     otp_token = forms.IntegerField(label=_("Token"), min_value=1,
                                    max_value=int('9' * totp_digits()))
+    remember = forms.BooleanField(required=False,
+                                  label=_("Remember this device for %s days")
```

Comment:
This won't produce the expected translation for some languages. See also [pluralization](https://docs.djangoproject.com/en/2.0/topics/i18n/translation/#pluralization) in the Django documentation.